roles/openshift_node/meta/main.yml

```diff
@@ -12,5 +12,6 @@ galaxy_info:
   categories:
   - cloud
 dependencies:
+- role: openshift_clock
```

Comment:
We'll probably want to coordinate this with @abutcher's PR here: https://github.com/openshift/openshift-ansible/pull/1501, since depending on which role goes first, the openshift_etcd role he introduces should have a dependency on openshift_clock as well.